index.html

```diff
@@ -0,0 +1,191 @@
+                    <li class="javascript">javascript</li>
+                </ul>
+                <button class="see-project" type="button">See Project</button>
+            </div>
+        </section>
+        <section class="work-samples">
+            <div class="sample-1">
+                <img src="./assets/images/Snapshoot-Portfolio-1.png" alt="image">
+                <h3 class="title">Multi Post Stories</h3>
+                <div class="details">
+                    <div class="name">CANOPY</div>
+                    <div class="type">Back End Dev</div>
+                    <div class="year">2015</div>
+                </div>
+                <P class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                <ul class="tools">
+                    <li class="html">html</li>
+                    <li class="css">css</li>
+                    <li class="javascript">javascript</li>
+                </ul>
+                <button class="see-project" type="button">See Project</button>
+            </div>
+        </section>
+        <section class="work-samples">
+            <div class="sample-1">
+                <img src="./assets/images/Snapshoot-Portfolio-2.png" alt="image">
+                <h3 class="title">Tonic</h3>
+                <div class="details">
+                    <div class="name">CANOPY</div>
+                    <div class="type">Back End Dev</div>
+                    <div class="year">2015</div>
+                </div>
+                <P class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                <ul class="tools">
+                    <li class="html">html</li>
+                    <li class="css">css</li>
+                    <li class="javascript">javascript</li>
+                </ul>
+                <button class="see-project" type="button">See Project</button>
+            </div>
+        </section>
+        <section class="work-samples">
+            <div class="sample-1">
+                <img src="./assets/images/Snapshoot-Portfolio-3.png" alt="image">
+                <h3 class="title">Tonic</h3>
+                <div class="details">
+                    <div class="name">CANOPY</div>
+                    <div class="type">Back End Dev</div>
+                    <div class="year">2015</div>
+                </div>
+                <P class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                <ul class="tools">
+                    <li class="html">html</li>
+                    <li class="css">css</li>
+                    <li class="javascript">javascript</li>
+                </ul>
+                <button class="see-project" type="button">See Project</button>
+            </div>
+        </section>
+        <section class="about-me">
+            <h1>About<br>
+                Myself
+            </h1>
```

Comment:
Please change this to be any heading from `h2` to `h6`. for example, `h2`. since the `h1` should only exist once in your page.